healthcheck.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const router = express.Router({});
+
+router.get('/', async (_req, res, _next) => {
+	const healthcheck = {
+		uptime: process.uptime(),
+		message: 'OK',
+		timestamp: Date.now()
+	};
+	try {
+		res.send(healthcheck);
+	} catch (e) {
+		healthcheck.message = e;
```

Comment:
It doesn't look like this is used

Reply:
you are right, I have pushed a change for this.